app/scripts/use-snow.js

```diff
@@ -12,16 +12,45 @@ Changing this code must be done cautiously to avoid breaking the app!
   const isWorker = !self.document;
   const msg =
     'Snow detected a new realm creation attempt in MetaMask. Performing scuttling on new realm.';
+
+  // eslint-disable-next-line no-undef
+  const chromeExtensionId = chrome && chrome.runtime && chrome.runtime.id;
```

Comment:
What's the importance of the `chromeExtensionId` below really? I think we can allow fetching images from the `chrome-extension` scheme for all types of browser extension entities we create

Reply:
addressed in b769e71328